extension.js

```diff
@@ -1,180 +0,0 @@
-/**
```

Comment:
This was not making Travis happy, also having a clean build on the CI would be better